test/module/iroha-cli/client_test.cpp

```diff
@@ -183,12 +184,21 @@ TEST_F(ClientServerTest, SendTxWhenStatelessInvalid) {
 
   ASSERT_EQ(iroha_cli::CliClient(ip, port).sendTx(shm_tx).answer,
             iroha_cli::CliClient::OK);
-  auto tx_hash = shm_tx.hash();
-  auto res = iroha_cli::CliClient(ip, port).getTxStatus(
-      shared_model::crypto::toBinaryString(tx_hash));
-  ASSERT_EQ(res.answer.tx_status(),
+  auto getAnswer = [&]() {
+    return iroha_cli::CliClient(ip, port)
+        .getTxStatus(shared_model::crypto::toBinaryString(shm_tx.hash()))
+        .answer;
+  };
+  decltype(getAnswer()) answer;
+  unsigned read_attempt_counter(status_read_attempts);
```

Comment:
better to use `auto` here. But if you want to state concrete type use platform-independent option: uint32_t, for example.